src/react-imgix.js

```diff
@@ -80,15 +102,35 @@ function buildSrc({
       const dpr3 = constructUrl(rawSrc, { ...srcOptions, dpr: 3 });
       srcSet = `${dpr2} 2x, ${dpr3} 3x`;
     } else {
+      let showARWarning = false;
       const buildSrcSetPair = targetWidth => {
-        const url = constructUrl(rawSrc, {
+        let urlParams = {
           ...srcOptions,
           width: targetWidth
-        });
+        };
+        const aspectRatioDecimal = aspectRatio && parseAspectRatio(aspectRatio);
```

Comment:
Following on comments on type checks in `parseAspectRatio`;
It seems to be sufficient to just pass `aspectRatio` into `parseAspectRatio` and throw warning if `false` was returned.